docs/src/pages/docs/Miscellaneous/issue_codes.mdx

```diff
@@ -144,8 +144,8 @@ that the username is typed correctly and exists in the database.
 ## Issue 1013
 
 ```
-The password provided when connecting to a database is not valid.
+Either the username or the password used are incorrect.
 ```
 
-The user provided a password that is incorrect. Please check that the
-password is typed correctly.
+Either the username provided does not exist or the password was written incorrectly. Please
+check that the username and were typed correctly.
```

Comment:
I merged issue 1013 already, so your PR is trying to replace it. You need to use 1014 now.